srcpkgs/clifm/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'clifm'
+pkgname=clifm
+version=1.11
+revision=1
+build_style=gnu-makefile
+make_install_args="MANDIR=/usr/share/man"
+makedepends="libcap-devel acl-devel file-devel readline-devel"
+short_desc="Clifm is a Command Line Interface File Manager"
+maintainer="Greg Beard <gmbeard@googlemail.com>"
+license="GPL-2.0-only"
```

Comment:
According to https://github.com/leo-arch/clifm#license, this should be `GPL-2.0-or-later`? (ping @leo-arch for confirmation)

Reply:
Yes, it is GPL2 or later.

Reply:
I didn't spot this initially. Thank you. Fixed.